sorts/pancake_sort.py

```diff
@@ -1,9 +1,25 @@
-"""Pancake Sort Algorithm."""
-# Only can reverse array from 0 to i
-
+"""
+This is a pure python implementation of the pancake sort algorithm
+For doctests run following command:
+python -m doctest -v pancake_sort.py
+or
+python3 -m doctest -v pancake_sort.py
```

Comment:
Please reverse the order to encourage users to try __python3__ first.

Reply:
Ok, sure.